src/grid/initial_conditions.cpp

```diff
@@ -1335,8 +1351,16 @@ void Grid3D::Clouds()
             #ifdef DE
             C.GasEnergy[id]  = p_cl/(gama-1.0);
             #endif
+
+            #ifdef SCALAR
+	    #ifdef BASIC_SCALAR
+            C.basic_scalar[id] = C.density[id]*0.3;
+	    #endif
+
+	    //TODO for Helena: Alwin merged this from dev but isn't sure if it is right. 
```

Comment:
@helenarichie I wasn't sure what I was doing when I merged in this line. Can you check to make sure that it's right or if it needs to be changed to access the dust field in particular?

Reply:
This has been resolved in my latest commit to Alwin's enum-dev branch! All tests for hydro and dust pass.